### DISCUSSION TITLE
What is this icon?

### DISCUSSION THREAD

**Discussion by unoukujou** [ImranR98/Obtainium#1733] [Q&A] [answered]
Some apps have this icon. What is that?

![20240717_imgpipe_0](https://github.com/user-attachments/assets/8195e84d-586d-4ce0-93b3-5be0008403f5)

**Comment by DwainZwerg** [comment]
It is used to reset the installation status.

**Reply by unoukujou** [reply]
Thanks but I don't understand what's the purpose? And why only some apps have it, others not?

**Reply by DwainZwerg** [reply]
Just apps with a pseudo version should have it.

**Reply by unoukujou** [reply]
OK. But still don't understand the purpose of this feature. What would anyone accomplish by using it?

**Reply by ImranR98** [reply]
https://github.com/ImranR98/Obtainium/wiki#version-detection

**Reply by ImranR98** [reply, marked as answer]
Last section:

> ...In such cases, Obtainium would not be able to detect that the app's real OS version has changed and so it would not update its internal records accordingly - you would need to manually correct the inconsistency.

The "reset install status" and "mark as updated" buttons let you do that.
